### ISSUE TITLE
Improve usability of cmdlets by eliminating redundant parameters

### ISSUE THREAD

**Issue by zionyx** [3shapeAS/dockerbuild-pwsh#51]
In the world of docker, the image name comprises of the full text that includes these elements:
* registry
* image name
* tag / digest
as `<registry>/<image name>/<tag / digest>`

But the parameters the cmdlets ask for, are 3 separate parameters as stated above.

To improve usability, I think we should let the users formulate the complete image names themselves, than to have this logic added in the cmdlets.

This will eliminate the need for 3 parameters, and streamline them into 1, similar to how docker cli used to accept.

In short, eliminate these parameters:
* `-Registry`
* `-ImageName`
* `-Tag`
